### PR TITLE
#36 describe data models according to ERD

### PR DESCRIPTION
**Pull request by Romanskyy**
Created piccolo app, made basic piccolo settings, and created models.

### CODE REVIEW

File: bot_app/tables.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from piccolo.table import Table
+from piccolo.columns.column_types import UUID, Integer, Varchar, Text, Timestamp, ForeignKey
+
+
+class User(Table, tablename='user_table'):
+    id = UUID(primary_key=True)
+    telegram_user_id = Integer()
```
[the remark this answers is not in the document]
Comment (Romanskyy) [reply bot_app/tables.py:7]: agree)!

File: bot_app/tables.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    telegram_language = Varchar()
+    user_name = Varchar()
+    first_name = Varchar()
+    last_name = Varchar()
+
+
+class ContextClass(Table, tablename='context_class'):
```
[the remark this answers is not in the document]
Comment (Romanskyy) [reply bot_app/tables.py:14]: What is/are the reason/reasons for your suggestion? Is it bad to have a table name? I am not for but see no reasons to be against)

[the remark this answers is not in the document]
Comment (YanaAnisimova) [reply bot_app/tables.py:14]: Extra code and confusing that the table is supposedly renamed.

[the remark this answers is not in the document]
Comment (Romanskyy) [reply bot_app/tables.py]: done, except for User/user. user is a protected name so we have to use <tablename>

File: bot_app/tables.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Romanskyy) [reply bot_app/tables.py]: done

**Review by YanaAnisimova — APPROVED** [review]

**Pull request merged**